src/components/DataTable/DataTableEmpty.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import classnames from "classnames"
+import DataTableBody from "./DataTableBody"
+import DataTableRow from "./DataTableRow"
+
+interface DataTableEmptyProps {
+  children?: React.ReactNode
+  className?: string
+}
+
+const DataTableEmpty: React.FC<DataTableEmptyProps> = ({
+  className,
+  children,
+}) => (
+  <DataTableBody className={classnames("datatable-empty-tbody", className)}>
+    <DataTableRow>
+      <td className="datatable-empty-tcol">
+        <span>{children}</span>
```

Comment:
```suggestion
        {children}
```

Lets not wrap this in span, it can be a message, but maybe its a react.js component with some buttons to handle empty table?